### PR TITLE
Permission Framework Project Area

### PR DESCRIPTION
**Pull request by vinzscam**
## Hey, I just made a Pull Request!

This PR introduces a new Project Area specifically dedicated to the core modules of the Permission Framework.
In addition to maintain the core Permission Framework's modules, the goal of the Project Area is to facilitate the seamless integration of plugins with the Permission framework.

### CODE REVIEW

File: OWNERS.md [hunk 1]
```diff
@@ -66,6 +66,23 @@ Scope: The Kubernetes plugin and the base it provides for other plugins to build
...
+| -------------------- | ------------ | --------------- | ---------------------------------------------- | ---------------- |
+| Ainhoa Larumbe       | Spotify      | Imaginary Goats | [ainhoaL](http://github.com/ainhoaL)           | ainhoa#8085      |
+| Claire Casey         | Spotify      | Imaginary Goats | [clairelcasey](http://github.com/clairelcasey) | clairecasey#2710 |
+| Eric Peterson        | Spotify      | Imaginary Goats | [iamEAP](http://github.com/iamEAP)             | iamEAP#3058      |
+| Harry Hogg           | Spotify      | Imaginary Goats | [HHogg](http://github.com/HHogg)               | simplex#3451     |
+| Joon Park            | Spotify      | Imaginary Goats | [Joonpark13](http://github.com/Joonpark13)     | Sixpool#5060     |
+| Mike Lewis           | Spotify      | Imaginary Goats | [mtlewis](http://github.com/mtlewis)           | mtlewis#3658     |
+| Tim Hansen           | Spotify      | Imaginary Goats | [timbonicus](http://github.com/timbonicus)     | timbonicus#6871  |
+| Vincenzo Scamporlino | Spotify      | Imaginary Goats | [vinzscam](http://github.com/vinzscam)         | vinzscam#6944    |
```
Comment (freben) [inline OWNERS.md:84]: 🧠 🐐

File: .github/labeler.yml [hunk 2]
```diff
@@ -21,7 +21,8 @@ microsite:
   - microsite/**/*
 auth:
   - plugins/auth-backend/**/*
-  - plugins/permission-*/**/*
   - packages/core-app-api/src/apis/implementations/auth/**/*
   - packages/core-app-api/src/lib/Auth*/**/*
   - packages/core-plugin-api/src/apis/definitions/auth.ts
+area:permissions:
```
[the remark this answers is not in the document]
Comment (freben) [reply .github/labeler.yml:27]: Agreed

**Review by freben — APPROVED** [review]
Approved besides the naming question

**Comment by Rugvip** [comment]
The label changes have now been made. Let's :shipit:! 🎉

**Pull request merged**